doc: Add a note to the SIM activation instructions

divipillai opened the pull request:
Add a note about accepting either EID or ICCID
to the SIM activation instructions in
Thingy:91 and nRF9160 user guides.

Also, added include (rst syntax) where the steps are same in 
Thingy:91 and nR9160 user guides.

Signed-off-by: divya pillai<divya.pillai@nordicsemi.no>

divipillai commented:
The doc changes were reviewed and approved by Cole hafner (through teams).

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8318
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

NordicBuilder commented:
### Integration test specification

| Test Module | File based changes | Manually selected | West overwrite |
| ----------- | :----------------: | :---------------: | :------------: |

Test spec set is empty, this is probabley due to no matching entry in [.github/test-spec.yml](https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml)


[Detailed information of selected test modules](https://jenkins-ncs.nordicsemi.no/job/latest/job/sub/job/test-fw-nrfconnect-nrf/job/master/25529/artifact/test-spec/test-spec-summary.yml/*view*/)

*Note: This message is automatically posted and updated by the CI*

jayteemo reviewed (approved):
approved, but please fix the `password your chose` wording

wiba-nordic commented on doc/nrf/ug_nrf9160_gs.rst:
```diff
@@ -217,23 +222,28 @@ To create an nRF Cloud account, complete the following steps:
-You can now log in with your email and the password you chose.
+You can now log in to `nRF Cloud`_ with your email and password.
 After logging in, you are taken to the dashboard view that displays your device count and service usage.
+Next, you need to activate the SIM card for the |DK|.
```
```suggestion
Next, you need to activate the SIM card you will use in the |DK|.
```
Current version can be interpreted like the SIM card needs to be activated specifically for the DK, which is not the case.

wiba-nordic commented on doc/nrf/ug_nrf9160_gs.rst:
```diff
@@ -192,20 +192,25 @@ If you experience any problems during the process, restart the Programmer app by
 
 .. _nrf9160_gs_connecting_dk_to_cloud:
 
-Connecting the DK to nRF Cloud
-******************************
+.. |DK| replace:: nRF9160 DK
```
You need to switch the order of the replace definition and the reference tag, it's currently breaking the build.

umapraseeda commented:
We could also add the 'Add LTE Device' screenshot (available in the DK guide) to the Thingy:91 guide too (before the explanations of Device ID and Ownership code).

wiba-nordic commented:
On that note, here's a more recent version of that screenshot. There's small changes that have been made, it seems.
![nrfcloud_add_lte_device](https://user-images.githubusercontent.com/83582261/183615392-1f033078-ad1e-4503-8c25-98b981c2069b.png)

divipillai commented:
> On that note, here's a more recent version of that screenshot. There's small changes that have been made, it seems. ![nrfcloud_add_lte_device](https://user-images.githubusercontent.com/83582261/183615392-1f033078-ad1e-4503-8c25-98b981c2069b.png)

I can add this image to both DK and thingy:91 user guides.

umapraseeda reviewed (approved):
LGTM!

divipillai commented:
@jhn-nordic : Added you as a reviewer to review the note ([doc/nrf/ug_thingy91_gsg.rst](https://github.com/nrfconnect/sdk-nrf/pull/8318/files#diff-90d0e4bb792f25c3e98c1eeb33e528b8d7ff1ea2413b4975771ffeb64ec4fcd0)) I have updated in the Thingy:91 user guide's [Update the modem firmware on the nRF9160 SiP](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8137/nrf/ug_thingy91_gsg.html#id40)[](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8137/nrf/ug_thingy91_gsg.html#update-the-modem-firmware-on-the-nrf9160-sip) section. 
As discussed, not adding this note to the "through USB" section because the flow differs when using USB or debug probe.
